pkg/podvolume/util.go

```diff
@@ -71,14 +96,73 @@ func GetVolumeBackupsForPod(podVolumeBackups []*velerov1api.PodVolumeBackup, pod
 			continue
 		}
 
-		volumes[pvb.Spec.Volume] = pvb.Status.SnapshotID
+		volumes[pvb.Spec.Volume] = VolumeBackupInfo{
+			SnapshotID:     pvb.Status.SnapshotID,
+			UploaderType:   getUploaderTypeOrDefault(pvb.Spec.UploaderType),
+			RepositoryType: getRepositoryType(pvb.Spec.UploaderType),
+		}
 	}
 
 	if len(volumes) > 0 {
 		return volumes
 	}
 
-	return getPodSnapshotAnnotations(pod)
+	fromAnnntation := getPodSnapshotAnnotations(pod)
```

Comment:
I see the comment of `getPodSnapshotAnnotations` says
```// Deprecated: we will stop using pod annotations to record restic snapshot IDs after they're taken,
// therefore we won't need to check if these annotations exist.

```
Could you double check if we still need this in v1.10?

Reply:
https://github.com/vmware-tanzu/velero/pull/1577

The deprecation message was added back in v1.0.1

Let's confirm with community, but I hope we don't need to keep this in v1.10.0

Reply:
OK, an issue https://github.com/vmware-tanzu/velero/issues/5277 is opened for this purpose, let's double check it and discuss it in the community.